adbnx_adapter/adbnx_adapter/arangoDB_networkx_adapter.py

```diff
@@ -20,7 +20,15 @@ def __init__(self, conn):
             user_name = conn["username"]
```

Comment:
Should we have defaults here as well?
localhost:8529 no auth?

Reply:
@joerg84 will check that with the next push.